Fix outdated labels (#24182)
mdn/content: files/en-us/mdn/community/mdn_content/index.md

To help you choose what content issues to work on, we've sorted them using GitHub labels.

The labels below help you find tasks based on estimated effort to complete the task.

| Label                                                                                                                                       | Description                                          |
| ------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small) | A task that will probably require little effort. |
| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium)       | A task that will probably require a significant effort. |

> | [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)      | A task that will probably require a lot of effort. |

Reviewer: I can't tell the semantic difference between "significant" and "a lot of". I suggest choosing something more nuanced for "effort: medium", because I don't want to get into the actual metrics.

Reviewer 2: Fair enough @Josh-Cena . I've updated to "non-trivial" - if you don't like that, propose an alternative :-)

Reviewer: I propose "medium" and "large" 😄 (Half joking, but it would not be a bad idea)

> | [Small-effort tasks on the content repo](https://github.com/mdn/content/labels/effort%3A%20small) | A task that will probably require a small effort. |
> | [Medium-effort tasks on the content repo](https://github.com/mdn/content/labels/effort%3A%20medium)       | A task that will probably require a medium effort.    |
> | [Large-effort tasks on the content repo](https://github.com/mdn/content/labels/effort%3A%20large)      | A task that will probably require a large effort.     |

Reviewer 2: Actually, the left column no longer contains labels. Should be:

```suggestion
| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small) | A task that will probably require little effort. |
| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium)       | A task that will probably require a significant effort. |
| [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)      | A task that will probably require a lot of effort. |
```

> | [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | A task that will probably require little effort. |
> | [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | A task that will probably require non-trivial effort. |
> | [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | A task that will probably require a lot of effort. |

Reviewer: ```suggestion
| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | A task that will probably require little effort, like adding one sentence. |
| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | A task that will probably require non-trivial effort, like updating content to reflect the spec. |
| [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | A task that will probably require significant and large-scale effort, like writing many new pages. |
```

> | Label                                                                      | Description                                          |
> | -------------------------------------------------------------------------- | ---------------------------------------------------- |
> | [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | A task that will probably require little effort, like adding one sentence. |
> | [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | A task that will probably require non-trivial effort, like updating content to reflect the spec. |
> | [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | A task that will probably require significant and large-scale effort, like writing many new pages. |

Reviewer: ```suggestion
| Label                                                                      | Description                                                                                        |
| -------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | A task that will probably require little effort, like adding one sentence.                         |
| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | A task that will probably require non-trivial effort, like updating content to reflect the spec.   |
| [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | A task that will probably require significant and large-scale effort, like writing many new pages. |
```

> | [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | A task that will probably require little effort, like adding one sentence.                         |
> | [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | A task that will probably require non-trivial effort, like updating content to reflect the spec.   |
> | [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | A task that will probably require significant and large-scale effort, like writing many new pages. |

Reviewer 2: @Josh-Cena W.r.t. to your previous suggestion about using "medium", I don't love it, because if you do that we might as well just have the labels and no comments at all. 

I prefer what you have done now. Here is a suggested modification "to my taste".

```suggestion
| [effort: small](https://github.com/mdn/content/labels/effort%3A%20small)   | A task that will probably require little effort, such as fixing a typo, or adding a few sentences of clarification. |
| [effort: medium](https://github.com/mdn/content/labels/effort%3A%20medium) | A task that will probably require non-trivial effort. For example, updating content to reflect changes to the formal specification.  |
| [effort: large](https://github.com/mdn/content/labels/effort%3A%20large)   | A task that will probably require significant and large-scale effort, such as writing new guide and reference pages. |
```
